src/openeo_gfmap/manager/job_manager.py

```diff
@@ -53,11 +90,14 @@ def __init__(
         post_job_action: Optional[Callable] = None,
         poll_sleep: int = 5,
         n_threads: int = 1,
-        post_job_params: dict = {},
         resume_postproc: bool = True,  # If we need to check for post-job actions that crashed
         restart_failed: bool = False,  # If we need to restart failed jobs
+        dynamic_max_jobs: bool = True,  # If we need to dynamically change the maximum number of parallel jobs
+        max_jobs_worktime: bool = 10,  # Maximum number of jobs to run in a given time
+        max_jobs: int = 20,  # Maximum number of jobs to run at the same time
```

Comment:
these distinctions here are a bit complex, especially for a public API. I am thinking how we can make this easier for  for external users. Especially if anyone in the US wants to make use of this code, the distinction made here won't make sense.

Ideally, we can dynamically adjust the maximum number of jobs based on real-time conditions, such as load, demand, or specific schedules for different regions.

However I doubt we can currently get the system load as a user...

Reply:
I believe this is a feature we should discuss and evaluate with the core team

Reply:
any thoughts @jdries?

Reply:
Indeed it is quite a complex feature. Initially it was written as a hotfix for the cluster overload issue but I'm not even using on my own applications it anymore.

I can remove this for now, and possibly this will be implemented in another PR?

Reply:
I would indeed remove it, and try to even solve it on the server side instead